src/components/atoms/Text/Text.tsx

```diff
@@ -6,27 +6,48 @@ import { colors, TColors } from '../../../constants/colors';
 
 export interface ITextProps extends HTMLAttributes<HTMLSpanElement> {
   /**
-   * The weight of the rendered text. Avoid the use of `semibold` as for now.
+   * The weight of the rendered text.
+   * @default 'regular'
    */
-  weight?: keyof typeof typography.weights;
+  weight?: 'regular' | 'bold';
```

Comment:
semiBold should be supported instead of bold.

Reply:
According to @Gianeo only these two for regular text:
> Normal + bold => body text, Semibold => underlined links
I asked about it on the private rebrand channel 📐